lib/pgBackRest/Common/Ini.pm

```diff
@@ -120,7 +129,105 @@ sub load
+    my $strContentCopy = fileStringRead($self->{strFileName} . INI_COPY_EXT, {bIgnoreMissing => true});
+    my $strContent = fileStringRead($self->{strFileName}, {bIgnoreMissing => true});
+
+    # If both exist then select an authoritative version
+    if (defined($strContent) && defined($strContentCopy))
+    {
+        my $oContentCopy = iniParse($strContentCopy, {bIgnoreInvalid => true});
+        my $oContent = iniParse($strContent, {bIgnoreInvalid => true});
+
+        # If both files parse then select an authoritative version
+        if (defined($oContent) && defined($oContentCopy))
+        {
+            $self->{oContent} = dclone($oContentCopy);
+            my $bContentCopyHeader = $self->headerCheck({bIgnoreInvalid => true});
+            $self->{oContent} = dclone($oContent);
+            my $bContentHeader = $self->headerCheck({bIgnoreInvalid => true});
+
+            # If both headers are valid then select an authoritative version
+            if ($bContentHeader && $bContentCopyHeader)
+            {
+                # If both files have the same checksum then they are identical
+                if ($oContent->{&INI_SECTION_BACKREST}{&INI_KEY_CHECKSUM} eq
+                    $oContentCopy->{&INI_SECTION_BACKREST}{&INI_KEY_CHECKSUM})
+                {
+                    $self->{oContent} = $oContent;
+                }
+                # Else use the sequence to determine the authoritative version
+                else
+                {
+                    # Use main if it has the largest sequence
+                    if ($oContent->{&INI_SECTION_BACKREST}{&INI_KEY_SEQUENCE} >
+                        $oContentCopy->{&INI_SECTION_BACKREST}{&INI_KEY_SEQUENCE})
+                    {
+                        $self->{oContent} = $oContent;
+                    }
+                    # Else use copy if it has the largest sequence
+                    elsif ($oContent->{&INI_SECTION_BACKREST}{&INI_KEY_SEQUENCE} <
+                           $oContentCopy->{&INI_SECTION_BACKREST}{&INI_KEY_SEQUENCE})
+                    {
+                        $self->{oContent} = $oContentCopy;
+                    }
+                    else
+                    {
+                        confess &log(ERROR,
+                            "$self->{strFileName} and $self->{strFileName}" . INI_COPY_EXT .
+                            ' have different checksums but the same sequence number, likely due to corruption', ERROR_CONFIG);
+                    }
+                }
+            }
+            # If neither header is valid then error on main
+            elsif (!$bContentHeader && !$bContentCopyHeader)
+            {
+                $self->{oContent} = $oContent;
+            }
+            # Else only one header is valid and must be considered authoritative
+            else
+            {
+                if (!$bContentHeader)
+                {
+                    $self->{oContent} = $oContentCopy;
+                }
```

Comment:
There is no else here - this is to assign the copy if the main header did not validate.